packages/web-components/examples/codesandbox/components/masthead/package.json

```diff
@@ -16,6 +16,7 @@
   "devDependencies": {
     "@babel/core": "^7.0.0-0",
     "carbon-components": "^10.36.0",
+    "lit-element": "^2.5.1",
```

Comment:
@emyarod should this be added? I thought we removed `lit-element` and `lit-html` as dev dependencies.

Reply:
@jeffchew yeah I'm not sure, but it was the only way I could get the code sandboxes to load properly after users reported not being able to use them

Reply:
@emyarod we should actually look into this because ideally our adopters shouldn't need to do this.

Reply:
Checking another random PR, building the codesandbox example actually seems to build fine: 
https://webcomponents-codesandbox.s3-web.us-east.cloud-object-storage.appdomain.cloud/deploy-previews/7827/masthead/index.html

I'm wondering if it's a CSB specific issue?

Reply:
yeah I thought it might be related to CSB's bundler but I'm not 100% sure on that

Reply:
And funny enough, this actually sounds more related to this which was opened by @tay1orjones:
https://github.com/codesandbox/codesandbox-client/issues/4456

Reply:
@emyarod since it sounds like a CSB  issue, perhaps we can leave this in but leave a note somewhere in the code sandbox example in a pretty obvious place that `lit-element` is actually not required but added to get around an issue in CSB itself

Reply:
@jeffchew I added a note in `index.js`, would that be enough or should we add a README to that example instead?

I imagine some other code sandboxes may be affected by this issue but I'm not sure which ones exactly yet

Reply:
I think should be fine where you put it, thank you!